myapp/app.js

```diff
@@ -0,0 +1,77 @@
+const mysql = require('mysql');
+const express = require('express');
+const session = require('express-session');
+const path = require('path');
+var connection  = require('./database/config.js');
+
+var registrationRouter = require('./routes/registration-route.js');
+var loginRouter = require('./routes/login-route.js');
+var dashboardRouter = require('./routes/dashboard-route.js');
+var logoutRouter = require('./routes/logout-route.js');
+
+const app = express();
+app.use(session({
+	secret: 'secret',
+	resave: true,
+	saveUninitialized: true
+}));
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(path.join(__dirname, 'public')));
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'ejs');
+
+
+app.use('/', registrationRouter);
+app.use('/', loginRouter);
+app.use('/', dashboardRouter);
+app.use('/', logoutRouter);
+
+app.get('/', function(request, response) {
+	response.sendFile(path.join(__dirname + '/login.html'));
```

Comment:
What you have done in "/home", can be here in this route. No need of "/home" route
Flow should be like,
- When we get "/", if a user is not logged in then show the login page, otherwise the home page